src/pages/editor/index.ts

```diff
@@ -159,35 +157,19 @@ document.addEventListener("DOMContentLoaded", () => {
     const extender = firstCategory.querySelector(".extender") as HTMLElement;
     if (icon && extender) {
         // Temp disable transitions
-        const transitionProperties = "width, margin-left";
         icon.style.transitionProperty = "none";
         extender.style.transitionProperty = "none";
```

Comment:
Could this be `transition` instead of `transitionProperty`? I just tested and it seems to work

```suggestion
        icon.style.transition = "none";
        extender.style.transition = "none";
```

Reply:
if it works then im fine with it